snowexsql/tables/instrument.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy import Column, Integer, String
+from .base import Base
+
+
+class Instrument(Base):
+    __tablename__ = 'instruments'
+    # auto created id
+    id = Column(Integer, primary_key=True)
```

Comment:
Link tables don't need a primary key.

See [this example](https://www.slingacademy.com/article/sqlalchemy-how-to-use-many-to-many-relationships/) for reference